packages/eos-components/src/components/PlayButton.vue

```diff
@@ -74,6 +74,10 @@ export default {
 <style lang="scss" scoped>
 @import '../styles.scss';
 
+.play-button {
+  white-space: nowrap;
```

Comment:
In Bootstrap there is a `text-nowrap` class that does the same, so you don't have to add additional CSS.

Reply:
That's true! I've updated the PR to use that class instead of adding the css.